charts/logging-operator/README.md

```diff
@@ -9,8 +9,7 @@ Logging operator for Kubernetes based on Fluentd and Fluentbit.
 ## TL;DR;
 
 ```bash
-helm repo add kube-logging https://kube-logging.github.io/helm-charts
-helm install --generate-name --wait kube-logging/logging-operator
+helm install --generate-name --wait oci://ghcr.io/kube-logging/helm-charts/logging-operator --version 0.0.0
```

Comment:
The `--version 0.0.0` part shouldn't be necessary after the next release. If you don't specify a version, Helm should pick the latest release.

Reply:
I've added it as a separate example using a bash variable to show that the user should set it